testsuite/shared/src/main/java/org/jboss/as/test/shared/LayersTestBase.java

```diff
@@ -83,7 +83,13 @@ public abstract class LayersTestBase {
      * Included in the return value of {@link #getExpectedUnusedInAllLayers()}
      * only when testing provisioning from the wildfly feature pack.
      */
-    public static final String[] NO_LAYER_WILDFLY = {};
+    public static final String[] NO_LAYER_WILDFLY = {
+            // Preview stability 'mvc-krazo' layer cannot be provisioned in OOTB standard wildfly
```

Comment:
When reading the comment, I thought at default standalone config. <perhaps adding a mention of layer.